packages/agent/src/utils/forest-schema/generator-fields.ts

```diff
@@ -111,11 +111,7 @@ export default class SchemaGeneratorFields {
 
     if (relation.type === 'OneToMany') {
       targetName = relation.originKeyTarget;
-      targetField = SchemaUtils.getColumn(
-        foreignCollection.schema,
-        targetName,
-        foreignCollection.name,
-      );
+      targetField = SchemaUtils.getColumn(collection.schema, targetName, collection.name);
```

Comment:
👀

Reply:
It was on foreignCollection previously.
```suggestion
      targetField = SchemaUtils.getColumn(foreignCollection.schema, targetName, foreignCollection.name);
```